code-challenges.js

```diff
@@ -11,7 +11,18 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
+const checkTemp = (temp) => {
+  if(temp1 <= 212){
+    return `${temp1} is below boiling point`
+  }
+  else if(temp2 >= 212){
+    return `${temp2} is above boiling point`
+  }
+  else if(temp3 === 212){
+    return `${temp3} is at boiling point`
+  }
```

Comment:
Just about there! Kind of does the job- your argument isnt actually being check against. You should be checking against `temp` but instead, you're hard coding the variables into the function. Additionally, if we look at how we are checking less than or greater than, you're using 'or equal to' which means, if you input 212, into the argument and the rest of the function was correct it would hit the first if statement and return "212 is below boiling point.".